ormolu.cabal

```diff
@@ -105,7 +105,8 @@ library
         Cabal >=3.6 && <3.7,
         aeson >=1.0 && <3.0,
         template-haskell,
-        th-lift-instances >=0.1 && <0.2
+        th-lift-instances >=0.1 && <0.2,
+        memoize >=1.1 && <1.2
```

Comment:
Can you try to use `MemoTrie` instead? It is more popular and is in Stackage.

Reply:
Will try, I keep forgetting to look at Stackage.